### PR TITLE
Minor edits to the core concepts page

### PR DESCRIPTION
**Pull request by narenst**
Switched from third person to second person. This was the only place third person was used, so it was a little off to read.

Also the overview titles seemed superflous, so I cut them.

Let me know if you have already thought about them, just figured it was easier to make a PR to discuss about.

### CODE REVIEW

File: docs/getstarted/core_concepts.md [hunk 1]
```diff
@@ -89,8 +84,9 @@ different datasets. When using FloydHub, datasets are always kept separate from
 code.
 
 ### Why keep datasets separate from code?
-A data scientist tweaks his/her code often during the process of creating a
-deep-learning model. However, he/she doesn't change the underlying data nearly
+
+A data scientist you tweak your code often during the process of creating a
```
Comment (narenst) [inline docs/getstarted/core_concepts.md]: Ah, this should be "As a data scientist ..."

Comment (mckayward) [reply docs/getstarted/core_concepts.md]: Good catch

**Comment by mckayward** [comment]
Yeah I thought about the overview titles. Without them I thought it looked a little weird, or like a user might be less likely to read on. But I didn't feel strongly about it, so I am open to scrapping them. You're right that they don't really do much

**Review by mckayward — APPROVED** [review]

**Comment by narenst** [comment]
Cool, the changes look good. This can be merged!

**Pull request merged**